src/preferences/colorpalettesettings.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include "preferences/usersettings.h"
+#include "util/color/colorpalette.h"
+
+class ColorPaletteSettings {
+  public:
+    explicit ColorPaletteSettings(UserSettingsPointer pConfig)
+            : m_pConfig(pConfig) {
+    }
+
+    ColorPalette getHotcueColorPalette() const;
+    void setHotcueColorPalette(const ColorPalette& colorPalette);
+
+    ColorPalette getTrackColorPalette() const;
+    void setTrackColorPalette(const ColorPalette& colorPalette);
+
+  private:
+    static const QString hotcueColorPaletteGroup;
```

Comment:
Naming: kHotcueColorPaletteGroup

If these constants are static private they don't need to appear in the header and could instead be hidden in an anonymous namespace